src/atoms/index.jsx

```diff
@@ -315,14 +315,27 @@ const atoms = [{
   examples: [
     {tunnels: [{cost:40}]},
     {bridges: [{cost:40}]},
-    {hexagons: [{cost:"80"}]},
-    {hexagons: [{cost:"threeve",width:44,fillColor:"yellow",strokeWidth:0}]},
-    {diamonds: [{cost:"+20"}]},
     {tunnelEntrances: [{percent:1}]},
     {tunnelEntrances: [{angle:120,percent:1,rotation:-60,color:"red"},
                        {angle:180,percent:1,color:"orange"},
                        {angle:240,percent:1,rotation:60,color:"yellow"}
                       ]}]
+},{
+  group: "Shapes",
+  examples: [
+    {diamonds: [{cost:"+20"}]},
```

Comment:
I know it's not here yet, and we can change this later, but I think for regular shapes we'd want just a `label` field, and that's it. Or just a `text` field? Whatever the name is, something generic and just a string. Then on the meaningful items `bridge` / `tunnel` we put a valid `cost` or `value` field with the associated `costFormat` and `valueFormat`. Make sense (for the future)?